libs/remix-ui/run-tab/src/lib/components/contractDropdownUI.tsx

```diff
@@ -133,12 +133,12 @@ export function ContractDropdownUI (props: ContractDropdownProps) {
     if (enable) {
       setContractOptions({
         disabled: false,
-        title: 'Select contract for Deploy or At Address.'
+        title: 'Select compiled a contract to deploy or to use with At Address.'
       })
     } else {
       setContractOptions({
         disabled: true,
-        title: loadType === 'sol' ? '⚠ Select and compile *.sol file to deploy or access a contract.' : '⚠ Selected *.abi file allows accessing contracts, select and compile *.sol file to deploy and access one.'
+        title: loadType === 'sol' ? '⚠ Select and compile *.sol file to deploy or access a contract.' : '⚠ When there is a compiled .sol file, the choice of contracts to deploy or to use with AtAddress is made here'
```

Comment:
I could not find a way to see this title. 

Still, warning sign should be removed and title for second condition should be updated

Reply:
This only comes up when there is no active .sol file in the editor nor an abi in the editor.
<img width="408" alt="image" src="https://user-images.githubusercontent.com/6050001/176208604-0a539b0b-76ac-4957-a328-946b421d7fee.png">

I removed the warning icons. I think the text is OK as I had updated it.  If you'd like an edit, please made a suggestion.